### PR TITLE
More dashboard

### PR DESCRIPTION
**Pull request by kcjerrell**
# Description

There's a bunch of stuff in here: loading indicators, message boxes/error messages, extra form stuff....

## Type of change

- [x] New feature (non-breaking change which adds functionality)

## Change Status

- [x] Complete, tested, ready to review and merge

# How Has This Been Tested?

- [x] `npm test`

# Checklist

- [x] My code follows the style guidelines of this project
- [x] I have performed a self-review of my own code
- [ ] My code has been reviewed by at least one peer
- [x] I have commented my code, particularly in hard-to-understand areas
- [x] I have made corresponding changes to the documentation
- [x] My changes generate no new warnings
- [x] New and existing unit tests pass locally with my changes
- [ ] There are no merge conflicts

### CODE REVIEW

File: src/components/AdminDashboard/AddIncident.js [hunk 1]
```diff
@@ -113,8 +112,6 @@ const AddIncident = props => {
   };
```
Comment (pflee1989) [inline src/components/AdminDashboard/AddIncident.js:113]: Not familiar with the code, but I trust that this could be merged soon. ^^

Comment (pflee1989) [reply src/components/AdminDashboard/AddIncident.js:113]: Hope we can get it all together tomorrow morning.

**Review by pflee1989 — APPROVED** [review]

**Review by andrewsbusby — APPROVED** [review]
It looks good to me.

**Review by lncpadawan — APPROVED** [review]
Important last-minute tweaks!

**Pull request merged**